src/Misc/expressionFunc/hashFiles/.eslintrc.json

```diff
@@ -1,6 +1,6 @@
 {
-    "plugins": ["jest", "@typescript-eslint"],
-    "extends": ["plugin:github/es6"],
```

Comment:
per docs we should pin to [recommended](https://github.com/github/eslint-plugin-github#setup)